README.md

```diff
@@ -12,6 +12,17 @@ also be discussed in the forum at https://forum.snapcraft.io.
 Our localization platform is Launchpad, translations are managed at
 https://translations.launchpad.net/ubuntu/+source/subiquity/
 
+To update translation template in launchpad:
+ * execute $ make i18n
+ * make an upload of subiquity into Ubuntu
+ * wait for it to migrate
```

Comment:
Is this actually required?  I would've thought launchpad imports translations from -proposed, not just from the release pocket.

Also I would make this explicit, "wait for it to migrate to the release pocket".

Reply:
If we ever do split publishing and build subiquity snap for a given lts series, then translations might need to come from the -updates pocket.

It was not obvious to me, at which point the translation templates are updated, as I see two uploads of translations per package =( once during -proposed and once during migration. And I was not around to request tarball export after it was published in -proposed but didn't migrate yet, to confirm that it is enough for it to just get published in -proposed. So i guess this is a precaution?

Reply:
OTOH, "wait for it to migrate" before doing what? this is the last thing in that section.  The next part is for translators to translate, which is a lot longer of a wait.  So maybe just omit the last line

Reply:
right, i split this into two lists. Maybe it should be the first item in the "export translations & commit them" checklist?